vsintegration/src/FSharp.Editor/Diagnostics/SimplifyNameDiagnosticAnalyzer.fs

```diff
@@ -18,7 +19,8 @@ open Microsoft.VisualStudio.FSharp.LanguageService
 
 type private TextVersionHash = int
 
-[<DiagnosticAnalyzer(FSharpConstants.FSharpLanguageName)>]
+// Disable until we work out how to make this low priority
+//[<DiagnosticAnalyzer(FSharpConstants.FSharpLanguageName)>]
```

Comment:
Yeah, I know.  But I feel it's the right thing to do until we get really authoritative information on https://github.com/dotnet/roslyn/issues/20390

SimplifyName is different to the other analyzers in that it makes many requests to FCS - one for each identifier